.env_template

```diff
@@ -1,2 +1,11 @@
-REACT_APP_EGGHEAD_BASE_URL=https://egghead.io/api/v1
-REACT_APP_EGGHEAD_JWT=YOUR_JWT_TOKEN_GOES_HERE
\ No newline at end of file
+# FAKE
+REACT_APP_EGGHEAD_BASE_URL=http://localhost:4000/api/v1
+REACT_APP_EGGHEAD_JWT=null
+
+# DEV
+# REACT_APP_EGGHEAD_BASE_URL=http://localhost:5000/api/v1
+# REACT_APP_EGGHEAD_JWT=YOUR_JWT_TOKEN_GOES_HERE
+
+# PROD
+# REACT_APP_EGGHEAD_BASE_URL=https://egghead.io/api/v1
+# REACT_APP_EGGHEAD_JWT=YOUR_JWT_TOKEN_GOES_HERE
```

Comment:
I would like to automate environment selection with a flag passed to the `yarn start` script instead of commenting out / manually replacing these values